src/handlers/movies/movieById.ts

```diff
@@ -0,0 +1,32 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import { notFoundResponse, serverErrorResponse } from '../../utils/api/apiResponses';
+import getMovieById from '../../models/Movie/getById';
+import { CONTENTFUL_INCLUDE } from '../../types/contentful';
+import { isCustomContentfulError } from '../../utils/api/utils';
+
+export const handler: APIGatewayProxyHandler = async (event) => {
+	const movieId = event?.pathParameters?.id;
+
+	try {
+		const movie = await getMovieById(movieId || '', {
```

Comment:
Rather than this pattern, you should check if movieId is null or undefined and return a 400 Bad Request early here.